### PR TITLE
chore: add responsible vouching person gh-teams maintainer

### PR DESCRIPTION
**Pull request by FaGru3n**
#### What would be change or will be added with this PR?
- add gh-teams maintainer as possible vouching person
#### Why do i want to change this?
- verification of new Members of Catena-X GitHub Organisation is unclear
#### Additional information
result from https://github.com/eclipse-tractusx/sig-infra/discussions/28

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/support-invite-member-to-github-org.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Who should invited to [Catena-X NG](https://github.com/catenax-ng)
 GitHub User:
 <!-- Please fill in the GITHUB-USER-ID -->
 
-Vouching person: <!-- Please fill in your Product Owners GITHUB-USER-ID-->
+Vouching person: <!-- Please fill in your GitHub-Teams Maintainer GITHUB-USER-ID or the Product Owners GITHUB-USER-ID -->
```
Comment (Siegfriedk) [inline .github/ISSUE_TEMPLATE/support-invite-member-to-github-org.md:15]: Which org? ;P

**Review by Siegfriedk — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/support-invite-member-to-github-org.md (hunk 1 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/support-invite-member-to-github-org.md]: [chore: added gh-org for gh-team maintainer](https://github.com/eclipse-tractusx/sig-infra/pull/45/commits/daa0a8766c78c9886c7142a695430502fc1ce4a9)

**Comment by FaGru3n** [comment]
hi @Siegfriedk feel free to check again.
adapted your information.

**Comment by Siegfriedk** [comment]
@FaGru3n great, just a nitpick: can you merge after making ng lower case?

**Comment by FaGru3n** [comment]
> @FaGru3n great, just a nitpick: can you merge after making ng lower case?

transformend NG -> ng 
[chore: ng instead of NG](https://github.com/eclipse-tractusx/sig-infra/pull/45/commits/9674a0cd9a5ece7749af03061715c3ccdf77507f)

**Pull request merged**